source/ruby-sass.html.haml

```diff
@@ -3,51 +3,35 @@ title: Ruby Sass
 ---
 
 %p.introduction
-  Ruby Sass was the original implementation of Sass, but it's since been
-  deprecated. It's currently in a sunset maintenance period, and it will be
-  officially unmaintained as of 26 March 2019. Ruby Sass users should migrate to
-  another implementation.
+
+  Ruby Sass was the original implementation of Sass, but it reached its end of
+  life as of 26 March 2019. It's no longer supported, and Ruby Sass users should
+  migrate to another implementation.
 
 %ul.list-columns
   %li
     :markdown
-      ### Maintenance Policy
-
-      Ruby Sass will continue to be maintained until 26 March 2019. During this
-      maintenance period, bugs will be fixed as much as possible but no new
-      features will be added. The only exception is for features that are
-      necessary to support new CSS syntax, which will continue to be added.
-
-      [Dart Sass][], the new primary implementation,
-      doesn't always behave 100% the same as Ruby Sass—it implements a number of
-      behavioral changes that were planned for Ruby Sass, but were never
-      implemented in a stable version. To make migration easier, recent versions
-      of Ruby Sass emit warnings for any code that won't be compatible with Dart
-      Sass. Ongoing maintenance includes adding any warnings for new behavioral
-      differences, if they come up.
-
-      [Dart Sass]: #{url_for "/dart-sass"}
+      ### But Why?
 
-      Once the maintenance period ends, Ruby Sass's repository will be archived,
-      no more changes of any kind will be made, and no new releases will be cut.
+      When Natalie and Hampton first created Sass in 2006, Ruby was the language
+      at the cutting edge of web development, the basis of their
+      already-successful [Haml][] templating language, and the language they
+      used most in their day-to-day work. Writing Sass in Ruby made it readily
+      available to their existing users and the whole booming Ruby ecosystem.
 
-      ### Adopt Ruby Sass
+      [Haml]: http://haml.info/
 
-      Support for Ruby Sass is being turned down because the Sass core team no
-      longer has time to maintain it as well as [Dart Sass][] and [LibSass][].
-      But that doesn't mean it has to go away. If a community member wanted to
-      take on the task of maintaining it, the core team would be happy to help
-      get them up to speed. It's not an easy job: it would require keeping on
-      top of bug fixes and implementing new language features as they're
-      designed by the language team. But it would also be an opportunity to work
-      on a project that's used by thousands of people every day.
+      Since then, Node.js has become ubiquitous for frontend tooling while Ruby
```

Comment:
Since we're mentioning Node here, does it make sense to remark that the Dart implementation makes itself available as a Node.js module, since JavaScript is a first-class compilation target for Dart?

Reply:
Done.